.golangci.yml

```
@@ -165,6 +165,10 @@ issues:
     - linters:
         - nolintlint
       source: "^//nolint:revive"
+    
+    - linters:
+        - nolintlint
+      source: "^?//nolint(:staticcheck)?"
```

Don't we need one for `bodyclose`, `nilerr` and `unparam` as well? :thinking: 

Also I think the regex shouldn't start with `^` if the comment if put at the end of a line right?

---

Oh I get it, can you change that source to

```suggestion
      source: "//nolint:[bodyclose|nilerr|staticcheck|unparama]"
```

And remove that single `//nolint` you found (it was a false positive from staticcheck I think)